uploadNeuroDB/tarchiveLoader

```diff
@@ -570,6 +570,10 @@ foreach my $minc (@minc_files) {
 ################################################################
 $utility->computeSNR($tarchiveInfo{TarchiveID}, $upload_id, $profile);
 ################################################################
+############### Compute DeepQC on T1W images ###################
+################################################################
+$utility->computeDeepQC($tarchiveInfo{TarchiveID}, $upload_id, $profile);
```

Comment:
technically, the call to this function should be optional based on whether DeepQC is allowed for the project. Maybe you could add a configuration in the imaging pipeline section "Compute Deep QC" with "yes/no" options. Then in the code, if the config is set to "yes" then call that computeDeepQC function.

Reply:
@cmadjar - good point!! will add that :sparkles:

Reply:
This should be split up into two calls, a POST to send the image to DeepQC, and a GET (not necessarily here) to get the results. If DeepQC goes down/never returns, it should not block LORIS forever.